rudder-webapp/debian/rules

```diff
@@ -120,8 +120,8 @@ binary-arch: install
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-doc pdf /usr/share/doc/rudder
 	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/rudder-doc html /usr/share/doc/rudder
 
-	# Install SELinux policies
-	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ rudder-webapp.pp /opt/rudder/share/selinux/
+	# Install SELinux policies -> Not done yet, only on RHEL
+	# dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ rudder-webapp.pp /opt/rudder/share/selinux/
```

Comment:
Yeah I thought about that too.

But since one day we will add the selinux policy (even on debian), so i thought it would be great just to uncomment that line

Removing it then